components/base/select/Select.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import ReactSelect, { components } from "react-select"; // https://bundlephobia.com/result?p=react-select@3.1.0
+import arrowSelectIcon from "../../../public/images/arrow-select-icon.svg";
+import DoubleArrowSelectIcon from "../../../public/images/double-arrows-select-icon";
+import CheckMarkIcon from "../../../public/images/check-mark-icon";
+import {
+  StyledDefaultDropdownIcon,
+  StyledOptionWrap,
+  StyledSelectedIconWrap,
+  defaultCustomStyles,
+  codeCustomStyles,
+  addAutoWidthStyles,
+} from "./Select.styles";
```

Comment:
Because all of the styles made this file quite long and hard to read, I ended up splitting them out. That said, I know our team approach to this was still an open question, so I'm happy to reconsider that decision!

Reply:
Yeah, I think I like it separated too. Let's keep it this way for now, and revisit the topic in our meeting next week.

Reply:
Yeah I like that you split them out here. I’m tempted to say “do what feels right” in situations like this cuz I don’t universally prefer one approach over the other?

When things are in one file I like that I can glance/swipe up to see the styles w/o opening another file, but at a certain point it’s easier to have 2 files open to avoid a bunch of scrolling/finding your place.